daemon/graphdriver/overlay2/overlay.go

```diff
@@ -159,6 +160,18 @@ func Init(home string, options []string, idMap idtools.IdentityMapping) (graphdr
 		return nil, overlayutils.ErrDTypeNotSupported("overlay2", backingFs)
 	}
 
+	usingMetacopy, err := doesMetacopy(testdir)
+	if err == nil {
+		if usingMetacopy {
+			logger.Debugf("overlay test mount indicated that metacopy is being used")
+		} else {
+			logger.Debugf("overlay test mount indicated that metacopy is not being used")
+		}
```

Comment:
Perhaps make this a single log where we print the value (could be using `WithField()` for that);

```go
logger.WithField("usingMetacopy", usingMetacopy).Debug("successfully detected meta-copy support")
```

Reply:
Ack